framework/addons/utils.py

```diff
@@ -8,48 +8,42 @@
     'none': 'danger',
 }
 
+CAPABILITY_SET = [
+    'Permissions',
+    'View / download file versions',
+    'Add / update files',
+    'Delete files',
+    'Logs',
+    'Forking',
+    'Registering'
+]
+
 def read_capabilities(filename):
 
-    lines = open(filename).readlines()
-    lines = [
-        line.rstrip().split('\t')
-        for line in lines
-    ]
+    data_file = open(filename, 'r')
+    data = load_json(data_file.read())
```

Comment:
You can use `json.load`, which takes a filepointer rather than a string.  Also, be sure to close the file pointer; you can use a with block:

```python
with open(filename, 'r') as data_file:
    data = json.load(data_file)
    addons = data['addons']
    # ...
```